spirv_msl.cpp

```diff
@@ -114,12 +105,13 @@ string CompilerMSL::compile()
 	backend.swizzle_is_function = false;
 	backend.shared_is_implied = false;
 	backend.native_row_major_matrix = false;
+	backend.flexible_member_array_supported = false;
 
 	uint32_t pass_count = 0;
 	do
 	{
 		if (pass_count >= 3)
```

Comment:
Do you need to change the branch as well, or?

Reply:
I'm not sure what you are asking here. What "branch" do you mean?

Reply:
if pass_count >= 3

Reply:
Oh...I see what you are commenting on...the change to the error text.

Hmmm...looks like a brain-farted and made that change incorrectly. I must have misread the >= as >. I've turned it back.